### ISSUE TITLE
example needs readme

### ISSUE THREAD

**Issue by cdaringe** [NogsMPLS/react-component-styleguide#4]
on installing and running the example, it wasn't clear where to go to see the styleguide.  there was a 3000 noted, so i browsed to that, but it rendered nothing

**Comment by NogsMPLS** [comment]
yeah, thanks. I kinda let this project slip as other things started happening. I'll see if I can pick some of this back up this month and get this project into working shape.

Pull Requests are welcome as well ;P
